docs/conf.py

```diff
@@ -172,7 +172,7 @@
 #html_file_suffix = None
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'pep8doc'
+htmlhelp_basename = 'pep8'
```

Comment:
Sure, but it's unrelated to fixing the formatting of the table and it doesn't have to be the same name.